docs/content/direct/core-chart-step-by-step-installation.md

```diff
@@ -0,0 +1,340 @@
+ITSes) that you just created (a host-type control plane is just an
+alias for the KubeFlex hosting cluster). You can do that with the
+following `kflex` commands; each creates a context and makes it the
+current one. See
+https://github.com/kubestellar/kubestellar/blob/0.28.0-alpha.2/docs/content/direct/core-chart.md#kubeconfig-files-and-contexts-for-control-planes
+for a way to do this without using `kflex`.
+Start by setting your current kubeconfig context to the one you used
+when installing this chart.
+
+kubectl config use-context $the_one_where_you_installed_this_chart
+kflex ctx --set-current-for-hosting # make sure the KubeFlex CLI's hidden state is right for what the Helm chart just did
+
+
+Finally, you can use `kflex ctx` to switch back to the kubeconfig
+context for your KubeFlex hosting cluster.
+```
+
+## Step 5: Verify Current Context
+
+check your current kubectl context:
+
+```bash
+kubectl config get-contexts
+```
+
+**Expected output (similar to):**
+```
+CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+*         kind-kubeflex   kind-kubeflex   kind-kubeflex   
+```
+
+## Step 6: Install KubeStellar Core with Control Planes
+
+now upgrade the chart to include the ITS and WDS control planes:
+
+```bash
+helm upgrade --install ks-core ./core-chart \
+--set "kubeflex-operator.hostContainer=kubeflex-control-plane" \
+--set "kubeflex-operator.externalPort=9443" \
+--set-json='ITSes=[{"name":"its1"}]' \
+--set-json='WDSes=[{"name":"wds1","ITSName":"its1"}]'
+```
+
+**Expected output (similar to):**
+```
+Release "ks-core" has been upgraded. Happy Helming!
+NAME: ks-core
+LAST DEPLOYED: Tue Jun  3 09:23:35 2025
+NAMESPACE: default
+STATUS: deployed
+REVISION: 2
+TEST SUITE: None
+NOTES:
+For your convenience you will probably want to add contexts to your
+kubeconfig named after the non-host-type control planes (WDSes and
+ITSes) that you just created (a host-type control plane is just an
+alias for the KubeFlex hosting cluster). You can do that with the
+following `kflex` commands; each creates a context and makes it the
+current one. See
+https://github.com/kubestellar/kubestellar/blob/0.28.0-alpha.2/docs/content/direct/core-chart.md#kubeconfig-files-and-contexts-for-control-planes
+for a way to do this without using `kflex`.
+Start by setting your current kubeconfig context to the one you used
+when installing this chart.
+
+kubectl config use-context $the_one_where_you_installed_this_chart
+kflex ctx --set-current-for-hosting # make sure the KubeFlex CLI's hidden state is right for what the Helm chart just did
+
+kflex ctx --overwrite-existing-context its1
+kflex ctx --overwrite-existing-context wds1
+
+Finally, you can use `kflex ctx` to switch back to the kubeconfig
+context for your KubeFlex hosting cluster.
+```
+
+## Step 7: Verify Control Planes
+
+check that the control planes have been created successfully:
+
+```bash
+kubectl get controlplane
```

Comment:
FYI, this is not wrong, but Kubernetes deliberately supports using the plural form of the resource here to make it read more naturally.